crates/rust-analyzer/src/config.rs

```diff
@@ -83,6 +83,8 @@ config_data! {
         cargo_useRustcWrapperForBuildScripts: bool = "true",
         /// Do not activate the `default` feature.
         cargo_noDefaultFeatures: bool    = "false",
+        /// Send progress notifications during project load.
+        cargo_reportLoadingProgress: bool = "true",
```

Comment:
I don't think this should be in the `cargo` category, these messages are our VFS loading files from disk, not Cargo